Formatting issue in db/schema.rb:14:1 on `make test`

## What is the current behavior?
When running `make test`,

```
Inspecting 252 files                                                                                                                                 
.....................................................................................................................................................
....................................................C..................................................                                              
                                                                                                                                                     
Offenses:                                                                                                                                            
                                                                                                                                                     
db/schema.rb:14:1: C: Layout/EmptyLinesAroundBlockBody: Extra empty line detected at block body beginning.                                           
                                                                                                                                                     
252 files inspected, 1 offense detected                                                                                                              
make: *** [test] Error 1
```

--- a/.rubocop.yml
+++ b/.rubocop.yml
@@ -1,5 +1,9 @@
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  Exclude:
+    - 'db/schema.rb'
+
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle, ConsistentQuotesInMultiline.
 # SupportedStyles: single_quotes, double_quotes
@@ -10,11 +14,6 @@ Style/StringLiterals:
 #turn off end of line checks, git will fix this
 EndOfLine:
   Enabled: false
-
-# Configuration parameters: CountComments, ExcludedMethods.
-Metrics/BlockLength:
-  Exclude:
-    - 'db/schema.rb'
 
 Metrics/ClassLength:
   Exclude:

--- a/db/schema.rb
+++ b/db/schema.rb
@@ -11,6 +11,7 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema.define(version: 20180711212702) do
+
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 

